.travis.yml

```diff
@@ -13,7 +13,7 @@ git:
 services:
   - docker
 script:
-  - make ci e2e status=keep
+  - make ci e2e status=keep globalnet=true
```

Comment:
This is just to test code, do not merge if this change still present.

Reply:
Biuld log: https://travis-ci.com/submariner-io/submariner-operator/builds/151055198